File: solutions/beecrowd/1008/1008.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`);

    val a: Int = sc.next().toInt();
    val b: Int = sc.next().toInt();
    val c: Double = sc.next().toDouble();

    println("NUMBER = %d".format(a));
    println("SALARY = U$ %.2f".format(b * c));
}

File: solutions/beecrowd/1008/1008.scala
object Main {
    def main(args: Array[String]) {
        val a = io.StdIn.readLine().toInt
        val b = io.StdIn.readLine().toInt
        val c = io.StdIn.readLine().toDouble

        println("NUMBER = " + (a))
        println("SALARY = U$ %.2f".format(b * c))
    }
}
